vision/setup.py

```diff
@@ -48,9 +48,10 @@ def create_version_file():
     'Pillow<=6.2.1',
     'numpy>=1.16.0',
     'matplotlib',
-    'gluoncv>=0.5.0,<1.0',
+    'gluoncv==0.9.0b20201125',
```

Comment:
What is the planned version of gluoncv we will officially be using for v0.1 release? I'd prefer to not depend on a pre-release build.

Reply:
v0.9.0, which will be released within a week